pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "nautobot-floor-plan"
-version = "1.0"
+version = "2.0.0rc2"
```

```suggestion
version = "2.0.0rc1"
```

We haven't shipped an RC1 of this app yet.

---

Fixed